pygfx/renderers/wgpu/_wgpurenderer.py

```diff
@@ -140,20 +141,26 @@ class WgpuRenderer(Renderer):
     also where fog is applied, as well as any custom post-processing
     effects.
 
+    Parameters:
+        target (WgpuCanvas or Texture): The target to render to, and what
+            determines the size of the render buffer.
+        pixel_ratio (float, optional): How large the physical size of the render
+            buffer is in relation to the target's physical size, for antialiasing.
+            See the corresponding property for details.
+        show_fps (bool): Whether to display the frames per second. Note that this
+            measure only makes sense if you're rendering as fast as you can and
+            if the canvas does not impose a frame rate limit.
```

Comment:
Well it also makes sense to see if you are dropping below such a limit.

Reply:
True. Will word it a bit differently.